### PR TITLE
New Presence: Broken Picturephone

### PR DESCRIPTION
**Pull request by luisgabriel53**

### CODE REVIEW

File: websites/B/Broken Picturephone/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "game",
+    "phone",
+    "broken",
+    "picturephone",
+    "telefone",
+    "sem",
+    "fio"
+  ],
+  "category": "games"
+};
```
Comment (Bas950) [inline websites/B/Broken Picturephone/dist/metadata.json:27]: bruh how can you have }; in json...

File: websites/B/Broken Picturephone/presence.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+presence.on("UpdateData", async () => {
+
+    const browsingStamp = Math.floor(Date.now() / 1000);
+
+    const button = document.getElementsByTagName('button');
+    const valor = button.length;
+    const players = document.getElementsByClassName('userActive');
+    const nump = `(${players.length} of 6 players)`;
+
+        const data: PresenceData = {
```
Comment (Bas950) [inline websites/B/Broken Picturephone/presence.ts:14]: merge consts

File: websites/B/Broken Picturephone/presence.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            largeImageKey: "large_image",
+            startTimestamp: browsingStamp;
+                };
+
+        if (valor == 1) {
+            data.details = "Creating a Room";
+            data.smallImageKey = "home";
+            data.smallImageText = "On homepage";
+        };
+        if (valor >= 6) {
```
Comment (Bas950) [inline websites/B/Broken Picturephone/presence.ts:24]: why does the if have ; at the end?

File: websites/B/Broken Picturephone/presence.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            data.details = "Playing";
+            data.state = nump;
+            data.smallImageKey = "playing";
+            data.smallImageText = "On game";
+            data.startTimestamp = browsingStamp
+        };
+        if (players.length > 6) {
```
Comment (Bas950) [inline websites/B/Broken Picturephone/presence.ts:31]: why does the if have ; at the end?

File: websites/B/Broken Picturephone/presence.ts [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+            data.state = "(6 of 6 players)";
+        };
+        if (data.details == null) {
```
Comment (Bas950) [inline websites/B/Broken Picturephone/presence.ts:34]: why does the if have ; at the end?

File: websites/B/Broken Picturephone/presence.ts [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+            presence.setTrayTitle();
+            presence.setActivity();
+        };
+        else {
+            presence.setActivity(data);
+        };
```
Comment (Bas950) [inline websites/B/Broken Picturephone/presence.ts:40]: why does the if have ; at the end?

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/B/Broken Picturephone/presence.ts (hunk 2 above)
Comment (luisgabriel53) [reply websites/B/Broken Picturephone/presence.ts:14]: how i do it? sorry i'm starting with ts

Comment (Bas950) [reply websites/B/Broken Picturephone/presence.ts:14]: const var1 = obj,
var2 = obj,
var3 = obj;

File: websites/B/Broken Picturephone/presence.ts [hunk 7]
```diff
@@ -4,16 +4,15 @@ const presence = new Presence({
...
-    const valor = button.length;
-    const players = document.getElementsByClassName('userActive');
-    const nump = `(${players.length} of 6 players)`;
+    const browsingStamp = Math.floor(Date.now() / 1000), 
+          button = document.getElementsByTagName('button'), 
+          valor = button.length, 
+          players = document.getElementsByClassName('userActive'), 
+          nump = `(${players.length} of 6 players)`; 
 
         const data: PresenceData = {
```
Comment (Bas950) [inline websites/B/Broken Picturephone/presence.ts]: const of data can also be merged with the above

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/B/Broken Picturephone/dist/metadata.json [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+    "name": "Luis Gabriel",
+    "id": "403648338286870529"
+  },
+  "url": "www.brokenpicturephone.com",
+  "description": {
+    "en": "A online game that you can draw, guess and send words to your friends guess."
+  },
+  "service": "Broken Picturephone",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/FYPmxU3.png",
```
Comment (Bas950) [inline websites/B/Broken Picturephone/dist/metadata.json:13]: please get a better quality logo, or use the one you use as large image key.

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload new proof of it working.

File: websites/B/Broken Picturephone/dist/metadata.json (hunk 8 above)
Comment (luisgabriel53) [reply websites/B/Broken Picturephone/dist/metadata.json:13]: Both are the same, do you want that I update them to a best quality?

Comment (Bas950) [reply websites/B/Broken Picturephone/dist/metadata.json:13]: image
I mean it looks really blurry so just something not blurry

**Comment by luisgabriel53** [comment]
> Upload new proof of it working.

image
image

**Review by Bas950 — APPROVED** [review]

**Review by Alanexei — APPROVED** [review]

**Pull request merged**